### DISCUSSION TITLE
Clock is wrong

### DISCUSSION THREAD

**Discussion by DasLixou** [qtile/qtile#3783] [Q&A] [answered]
My clock is wrong. uses wrong time zone. Tried to set it with the timezone parameter to `Europe/Berlin` but now it says `N/A`

**Comment by elParaguayo** [comment]
Have you installed `pytz` or `dateutil`?

**Reply by elParaguayo** [reply]
Also, have you set the timezone on your computer? You shouldn't need to set it in the clock unless you're planning to show a different timezone to the system.

https://wiki.archlinux.org/title/System_time#Time_zone

**Reply by DasLixou** [reply]
timedatectl says it is Europe/Berlin. i have installed python-dateutil via pacman.

**Reply by DasLixou** [reply]
i want Europe/Berlin but the time it shows is 2 hours more

**Reply by DasLixou** [reply]
so here (Europe/Berlin) it is 20:41 my timezone is set to Europe/Berlin but it still says 22:41

**Reply by elParaguayo** [reply]
It your system timezone is correct, you shouldn't need to set `timezone` in the the widget. However, if you set the timezone _after_ starting qtile then you'll need to restart qtile for the change to take effect.

**Reply by DasLixou** [reply]
No I even restarted qtile 20 times. But I can’t properly read the timedatectl, it shows a UTC and a CEST time and I don’t know what is currently set. But it says that it is Europe/Berlin. I’ll read through the logs and look if I find smth

**Reply by DasLixou** [reply]
Oh gosh I think it is somethink with arch Linux. The time zone is correctly, but even my git commits we’re committed „in 1 hour“ 😅 but why is it then saying Berlin. Any idea?

**Reply by elParaguayo** [reply]
For me, the first line of the timedatectl output is `Local Time`. What does yours say?

**Reply by DasLixou** [reply]
```
Local time: Mo 2022-08-22 12:44:14 CEST
Universal time: Mo 2022-08-22 10:44:14 UTC
RTC time: Mo 2022-08-22 10:44:14
Time zone: Europe/Berlin (CEST, +0200)
System clock synchronized: no
NTP service: inactive
RTC in local TZ: no
```

so here 10:44 is what i want but local says 12:44. Thats weird because 10:44 is the right time and it says it is Europe/Berlin.

**Reply by DasLixou** [reply]
it should be right, because it says it is CEST, aka. UTC +2 but local time displays something different..

**Reply by DasLixou** [reply]
:o i think i know whats happening. i ran `sudo hwclock --debug` and it says that it thinks my hardware clock is utc and says 10:55 and adds there 2 more hours. how can i make it clear that my hardware clock is cest not utc?

**Comment by DasLixou** [comment, marked as answer]
Ok got it. its a problem when dualbooting with windows. you can set it in arch via `sudo timedatectl set-local-rtc 1 --adjust-system-clock` so it knows that windows set the hardware clock to (in my case CEST) and now use CEST as CEST and not CEST as UTC :)
